library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608Parser.java

```diff
@@ -153,6 +156,12 @@
         continue;
       }
 
+      // Ignore repeated control characters.
+      if (ccData1 == previousCtrl1 && ccData2 == previousCtrl2) {
```

Comment:
I tried to read everything publicly available about EIA608. Correct me if I am wrong, but as far as I understand, repeated **Preamble Address Code**s will not affect the output, any formatting/styling/positioning that is higher priority than the ones before should simply override the previous setting.

Repeated **Mid Row Command**s similarly override previous lower priority commands, but they also should move the cursor (effectively printing a transparent space character to the output). So we might miss space characters when dropping something intentional. (Duplication of commands seemed to be optional, isn't it? If it ever makes sense and want to repeat a command, do they have to send it 4 times?)

If we repeatedly change a state based on an incoming **Misc Code** we might incorrectly handle line breaks, but maybeAppendNewline() probably handles most of the possible situations correctly by dropping a second new line character. 

If we do not care about white space errors, than we might be good, the only thing that should be handled carefully seems to be the **Backspace** command. Unfortunately, I do not have sample streams to check how they usually send them.